### DISCUSSION TITLE
VLAN Tag IDs in SIEM

### DISCUSSION THREAD

**Discussion by virginiait** [Security-Onion-Solutions/securityonion#7747] [Unsupported Versions] [answered]
We've an on-prem Standalone production install of Security Onion 2.3.100 (via ISO method) monitoring multiple network segments having VLAN Tag IDs. All stats show "ok". Running great. No packet drops. MTU up'd to 1550 on each monitoring interface.

Currently having difficulty getting logged VLAN Tag ID data to show up in the onboard ELK SIEM and/or an external Splunk SIEM (via Splunk Forwarder). SOD Rel 2.3.100 notes that while Zeek and Stenographer would still analyze traffic on a VLAN, they won't log the actual VLAN tag ID due to the way AF-PACKET works.

During initial setup process, we were prompted to select which tool would be used to generate metadata. The Zeek option was selected. The comments within the same step, stated Suricate will still run to generate NIDS alerts.

Is there an alternate method, tweak, or utility to get all VLAN tag IDs (whether associated with an alert or not) into either SIEM for analytics - without disabling Zeek, Stenographer, or Suricata?

Thanks in advance.

**Comment by dougburks** [comment, marked as answer]
One option might be to move to a distributed deployment with a separate forward node for each VLAN tag and set the hostname of the node based on the VLAN tag. This should allow you to search and filter based on VLAN tag.
